packages/scandipwa/src/component/CheckoutShipping/CheckoutShipping.component.js

```diff
@@ -42,12 +42,13 @@ export class CheckoutShipping extends PureComponent {
     renderOrderTotal() {
         const {
             totals: {
-                grand_total,
+                subtotal_with_discount,
+                tax_amount,
                 quote_currency_code
             }
         } = this.props;
 
-        const orderTotal = formatPrice(grand_total, quote_currency_code);
+        const orderTotal = formatPrice(subtotal_with_discount + tax_amount, quote_currency_code);
```

Comment:
Is it possible to fetch this data from BE, not to calculate on FE? if it is possible - it would be a better option than to perform this calculation here.